test/lib/analyzer/index.test.ts

```diff
@@ -173,12 +229,31 @@ test("analyzer", async (t) => {
   });
 
   for (const targetImage of Object.keys(examples)) {
-    const example = examples[targetImage];
-    const expectation = JSON.parse(
-      readOsFixtureFile(example.dir, "analyzer-expect.json"),
-    );
+    await t.test(targetImage, async (t) => {
+      const example = examples[targetImage];
+
+      // fileSyncStub.returns({
+      //   name: getOsFixturePath(example.dir, "image.tar"),
+      //   removeCallback: () => true,
+      // });
+      const actual = await analyzer.analyze(targetImage);
+
+      // if (!fs.existsSync(getOsFixturePath(example.dir, "analyzer-expect.json"))) {
+      //   fs.writeFileSync(
+      //     getOsFixturePath(example.dir, "analyzer-expect.json"),
+      //     JSON.stringify(actual, (k,v) => { return v === undefined ? null : v;}),
+      //   );
+      // }
+      const expectation = JSON.parse(
+        readOsFixtureFile(example.dir, "analyzer-expect.json"),
+      );
+
+      // // tslint:disable-next-line
+      // console.error(actual.results[2]);
+      // // tslint:disable-next-line
+      // console.error(expectation.results[2]);
```

Comment:
Can we remove all this commented out code please?

Reply:
Done